app/assess/models/ui/applicants_response.py

```diff
@@ -192,7 +193,8 @@ def _ui_component_from_factory(item: dict):
 
     elif presentation_type == "file":
         # TODO(FS-2065(?)): add href as link to download actual file?
-        return AboveQuestionAnswerPairHref.from_dict(item, href=answer)
+        presighned_url = get_file_url(filename=answer, application_id=application_id)
```

Comment:
Small type on 'presighned' too.  Also if filename is None, should we be creating an AboveQuestionAnswerPair instead of an AboveQuestionAnswerPairHref?  Nothing to link to if it's None

Reply:
Actually.  This is already handled now I re-read the code... my bad